test/array-test.cpp

```diff
@@ -121,4 +121,4 @@ void array_test()
 	for (std::size_t idx = 0; idx < m.size() && swapTest; ++idx)
```

Comment:
```suggestion
	for (std::size_t idx{ 0 }; idx < m.size() && swapTest; ++idx)
```

Reply:
In general, traditionally people don't use UIS in for loop initialization, mostly because it makes it really hard to read code. However, IMO it is better to use them there as well. 

That said, the proposed change to `array-cpp` is not aligned with the goals of this PR. Yet, I would normally accept the suggestion, but I see that file needs many other (related) changes as well. So, I propose to delay making the change so an not to unnecessarily pollute the change list for this PR.